Core/include/main.h

```diff
@@ -1,4 +1,4 @@
 #pragma once
 #include "../include/PPM.h"
-#include "Vec4/Vec4.h"
-#include <iostream>
\ No newline at end of file
+#include <iostream>
+#include "../include/Ray.h"
```

Comment:
Recommend adding a proper include path and using exclusively `include <path/to/header.hpp>` everywhere.

Reply:
My CMake skills are really limited so I'm not sure how to do this using CMake only

Reply:
`target_include_directories(foo PRIVATE include src)` should be enough